spec/facility_spec.rb

```diff
@@ -22,5 +27,82 @@
       @facility.add_service('Vehicle Registration')
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
+
+    it '#has to have the service in order to do the function' do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      expect(registrant_1.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+      expect(registrant_1.permit?).to eq(true)
+      expect(@facility_1.administer_written_test(registrant_1)).to eq(false)
+      @facility_1.add_service('Written Test')
+      @facility_1.administer_written_test(registrant_1)
```

Comment:
Where is the assertion that this is now true? You have line 35 that shows me it's false, then you add the service, so I'd want to see an assertion that shows me this is now working and returning true or has the desired side effects.